chrome-2023-experiment-overview.md

```diff
@@ -0,0 +1,101 @@
+# Chrome 2023 Experiment Overview
+
+In early 2023, Google Chrome will be running an origin trial to allow developers to try out several new enhancements related to speculation rules and prefetching. These enhancements are focused around allowing the browser to automatically assist with prefetching with minimal developer work, and allowing developers to measure when their pages are successfully prefetched.
```

Comment:
I think much of this stuff works for prerendering too, right? Should we use "preloading" here and throughout?